packages/plugin-table/src/Table.ts

```diff
@@ -5,8 +5,136 @@ import { TableCellDomParser } from './TableCellDomParser';
+        },
+        addColumnBefore: {
+            handler: this.addColumnBefore.bind(this),
+        },
+        addColumnAfter: {
+            handler: this.addColumnAfter.bind(this),
+        },
+        mergeCells: {
+            handler: this.mergeCells.bind(this),
+        },
+        unmergeCells: {
+            handler: this.unmergeCells.bind(this),
+        },
+    };
+    shortcuts = [
+        {
+            pattern: 'CTRL+<Slash>',
+            commandId: 'addRowBelow',
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Slash>',
+            commandId: 'addRowAbove',
+        },
+        {
+            pattern: 'CTRL+<Equal>',
+            commandId: 'addColumnAfter',
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Equal>',
+            commandId: 'addColumnBefore',
+        },
+        {
+            pattern: 'CTRL+<Period>',
+            commandId: 'mergeCells',
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Period>',
+            commandId: 'unmergeCells',
+        },
+    ];
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Add a row above the cell at given range start.
+     *
+     * @param params
+     */
+    addRowAbove(params: CommandParams): void {
+        const range = params.context.range;
+        const cell = range.start.ancestor(TableCellNode);
+        if (!cell) return;
+
+        cell.table.addRowAbove(cell);
+    }
+    /**
+     * Add a row below the cell at given range start.
+     *
+     * @param params
+     */
+    addRowBelow(params: CommandParams): void {
+        const range = params.context.range;
+        const row = range.start.ancestor(TableCellNode);
+        if (!row) return;
+
+        row.table.addRowBelow(row);
+    }
+    /**
+     * Add a column before the cell at given range start.
+     *
+     * @param params
+     */
+    addColumnBefore(params: CommandParams): void {
+        const range = params.context.range;
+        const cell = range.start.ancestor(TableCellNode);
+        if (!cell) return;
+
+        cell.table.addColumnBefore(cell);
+    }
+    /**
+     * Add a column after the cell at given range start.
+     *
+     * @param params
+     */
+    addColumnAfter(params: CommandParams): void {
+        const range = params.context.range;
+        const cell = range.start.ancestor(TableCellNode);
+        if (!cell) return;
+
+        cell.table.addColumnAfter(cell);
+    }
+    /**
+     * Merge the cells at given range into the first cell at given range.
+     *
+     * @param params
+     */
+    mergeCells(params: CommandParams): void {
+        const range = params.context.range;
+        const cells = range.targetedNodes(TableCellNode);
+        const managerCell = cells.shift();
+        for (const cell of cells) {
+            cell.managerCell = managerCell;
+        }
```

Comment:
I think a proper function would be more obvious, something like `setManagerCell` ? Or maybe override `mergeWith` to have a different behavior if the given node is a table cell ?